litellm_sync_backend.py

```
@@ -207,3 +209,83 @@ def generate(self, request: SyncGenerationRequest) -> SyncGenerationResult:
+            )
+        kwargs: Dict[str, Any] = {
+            "model": request.model,
+            "messages": _messages(request.contents, config),
+        }
+        api_key = self._api_key
+        if not api_key:
+            try:
+                from litellm_provider_config import load_provider_api_key
+
+                api_key = load_provider_api_key(provider_from_model(request.model))
+            except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)